demonstrations/tutorial_shadow_hamiltonian_simulation.py

```diff
@@ -0,0 +1,337 @@
+r"""Shadow Hamiltonian Simulation
+=================================
+
+.. figure:: ../_static/demonstration_assets/shadow_hamiltonian_simulation/shadow_hamiltonian_simulation_opengraph.png
+    :align: center
+    :width: 35%
+
+Introduction
+------------
+
+Shadow Hamiltonian simulation is a new approach to quantum simulation on quantum computers [#SommaShadow]_.
+Despite its name, it has little to do with :doc:`classical shadows </demos/tutorial_diffable_shadows>`.
+In quantum simulation, the goal is typically to simulate the time evolution of expectation values
+of :math:`M` observables :math:`O_m,` for :math:`m=1,\ldots ,M,`.
+The common approach is to evolve the wave function :math:`|\psi\rangle` and then measure the desired observables after the evolution.
+
+In shadow Hamiltonian simulation, we instead directly encode the expectation values in a proxy state — the **shadow state** — 
+and evolve that state accordingly. Specifically for time evolution, we can write a shadow Schrödinger equation that governs the
+dynamics of the shadow state.
+
+This is fundamentally different to the common approach. Foremost, the dimensionality of the 
+shadow system no longer depends on the number of constituents :math:`n` of the system. In fact, the underlying state can be mixed or even infinite dimensional.
```

Comment:
```suggestion
shadow system no longer depends on the number of constituents, :math:`n`, of the system. In fact, the underlying state can be mixed or even infinite-dimensional.
```
Or put the n at the end of the sentence, after a comma.
Also, either hyphen or use 'infinitely'.